### ISSUE TITLE
ESSENTIAL TWEAKS Task Manager Freeze

### ISSUE THREAD

**Issue by jimkal906** [ChrisTitusTech/win10script#275]
After the apply of Essential Tweaks without Restart Task Manager unable to close fast. It will freeze for 3-4 seconds and then it will close
![Screenshot 2022-01-26 032908](https://user-images.githubusercontent.com/71830530/151087871-54743482-3e2a-4609-bd68-a6e2e97430c4.png)
.

**Comment by Carterpersall** [comment]
Without knowing if this is resolved via a restart there is not much that can be done to resolve this. If it is fixed after a restart, then nothing further needs to be done. If it is not resolved after a restart, further research will need to be performed.

**Comment by jimkal906** [comment]
It is not resolved after a restart. You need to do a research on that. I had the same problem on 3pc devices fresh install windows 10.
Thank you if you have the time to do your research 😊

**Comment by Carterpersall** [comment]
In a hyper-v windows 11 VM I was unable to confirm the behavior. It's possible though that the issue is windows 10 specific but is unlikely.

**Comment by jimkal906** [comment]
On windows 10 try it.  Windows 11 maybe its okey. Try it on vmware or any other virtulization programm😊

**Comment by Carterpersall** [comment]
In a Windows 10 Hyper-V (Same Hypervisor as Windows Sandbox) VM I was unable to replicate the problem. Can I have your windows version and computer specs? This might be coming from a performance issue with task manager with details enabled (even my beefy computer takes a few seconds to open the app).

**Comment by jimkal906** [comment]
[image: image.png]
I hope you found the solution behind this . Btw i already run a debloat
script by farag so i don't know if the Chris script don't go well with
Farag script.
i test it on a sandbox and virtual and this is the specs of the v-machine
[image: image.png]

**Comment by Carterpersall** [comment]
It doesn't seem like the pictures you sent are showing up unfortunately, it might have something to do with you sending them over email. You might have to upload them to GitHub directly for them to show up.

**Comment by jimkal906** [comment]
![Uploading Screenshot_20220326-101520.png…]()
Try again and i hope its okey

**Comment by jimkal906** [comment]
Sorry for the late response. 
![imagae](https://user-images.githubusercontent.com/71830530/160287646-33d274d0-0955-4ff8-9606-3fff46516b3e.png)
![image](https://user-images.githubusercontent.com/71830530/160287649-2b259ca5-7c4f-4cb8-9e43-9a1be9b354a8.png)

Here the images and the information you asked😀

**Comment by Carterpersall** [comment]
It seems that your system is plenty fast enough(it is almost exactly what I have in my pc). From what I could find online it seems that it could be tied to the system drive being slow. So if you have a hard drive as your system drive, that could be your issue. If you don't, this issue could be just Windows being it's normal annoying self.
Either way, I'd recommend running these three commands to check for system corruption. It likely won't fix the issue but it's easy to do so it's worth a shot:
1: sfc /scannow
2: DISM /Online /Cleanup-Image /RestoreHealth   (You can add on /StartComponentCleanup if you want to clear some storage space while your at it)
3: Restart and then run sfc /scannow one more time
After those, restart again
It's good to run those every once in a while anyway as it has no real chance of breaking anything and fixes most minor Windows system corruption.

**Comment by jimkal906** [comment]
For storage i have 1nvme and 2ssd all new. The 3commands i do it every
2weeks to be exact . And the problem still exists. On enterpraise windows
10 i cant do it bc its sandbox so if i close it it will destroy all the
current data of virtual machine.

**Comment by jimkal906** [comment]
If i open the script via notepad++ i did it most of the things manual. So
maybe its the task manager thing. Dunno

**Comment by Carterpersall** [comment]
Yeah then I don't think I can help you very much unfortunately. It seems that your issue is just Windows being it's annoying self. I guess at least if you upgrade to Windows 11 the new task manager that is coming in 22H2 would probably fix that issue.

**Comment by Brandonbr1** [comment]
for now use Process Explorer https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer

**Comment by DidakaxD** [comment]
same issue here on Windows 10 Pro. Have you found a solution?
